cmd/device/device.go

```diff
@@ -15,10 +15,11 @@
 package device
 
 import (
-	adddevice "github.com/edgexfoundry-holding/edgex-cli/cmd/device/add"
-	listdevice "github.com/edgexfoundry-holding/edgex-cli/cmd/device/list"
-	rmdevice "github.com/edgexfoundry-holding/edgex-cli/cmd/device/rm"
-
+	"github.com/edgexfoundry-holding/edgex-cli/cmd/device/add"
```

Comment:
time to remove -holding ? :-)

Reply:
The current PR depends on #271 and this is why the change from both PRs are visible here.
Better approach probably was to wait the first PR to be merged.

Reply:
good catch up! I haven't noticed it at all.
It looks to me that moving out of holding haven't happened entirely correctly.
@jpwhitemn any ideas

Reply:
LF just does a move of the code for us.  If there are any code references, then there has to be work to make those changes in the code base once it is moved.

Reply:
I created separe issue about this 
https://github.com/edgexfoundry/edgex-cli/issues/278